frontend/src/components/navbar/Navbar.styled.ts

```diff
@@ -3,6 +3,7 @@ import styled from "styled-components";
 const NavbarWrapper = styled.nav`
     position: fixed;
     display: flex;
+    z-index: 10;
```

Comment:
why not 1 or 2?

If u try to force some changes, because have a unexpected behavior, that's wrong way solider :)

Reply:
z-index 1 and 2 are already being used by other ui components. 10 is a value which comes from assumption that there will never be any ui component that will display over NavbarWrapper. thus there is still space to create future ui items with z-index of 3,4,5 etc.

Reply:
yes, that was my intent. at this point (and presumably also in the future) z-index for this component will consistently need to stay at the very top. i refactored it to z-index: 5.